remove simple, easily and other words from documentation (#854)
jhipster/jhipster.github.io: pages/jhipster_uml.md

***

# <a name="howtouse"></a>How to use it

> JHipster-UML is not hard to use, you only need a class diagram exported in XMI and JHipster-UML will parse it to create your entities.

Reviewer: Is there another way to say it without using hard ? Using intuitive for instance ? Or maybe just remove the first part of the sentence ?

Author: In fact this was one of tricky parts. Will update it. Thanks

JHipster and JHipster UML support both support enumerations.
Here's how you define them:
  - For Modelio, drag and drop the `Enumeration` object and place it somewhere. Finally, add the `Enumeration Literal` object to the enum to add it;

>   - For UML Designer, there is the `Enumeration` object that can be placed and used. However, the literal is not called `Enumeration Literal` but on;y `Literal`;

Reviewer: typo here
```suggestion
  - For UML Designer, there is the `Enumeration` object that can be placed and used. However, the literal is not called `Enumeration Literal` but only `Literal`;
```

## <a name="parsermodifications"></a>Parser modifications

> The 1.0.0 release brings a new parser system making any change (parser creation, update, deletion) hazzle-free, provided the XMI can be parsed.

Reviewer: Do you mean:
```suggestion
The 1.0.0 release brings a new parser system making any change (parser creation, update, deletion) hassle-free, provided the XMI can be parsed.
```
